test/conftest.py

```diff
@@ -41,3 +50,26 @@ def handled_workspace(generated_workspace):
     yield generated_workspace
 
     delete_workspace(generated_workspace)
+
+
+@pytest.fixture
+def populated_workspace(
+    handled_workspace, data_directory, server
+) -> Tuple[str, Tuple[List[str], List[str]]]:
+    """
+    Populate a workspace with some data.
+
+    Returns a Nested Tuple of the form:
+    `(workspace_name: str, (graphs: List[str], tables: List[str]))`
```

Comment:
Why a Tuple? A TypedDict would generally be preferable.

Reply:
I thought a tuple felt like a more natural way to return this, plus it allows for argument destructuring, like done here: https://github.com/multinet-app/multinet-server/blob/45f96accf551dd743af71233d9dc8c0f9412b2d6/test/test_aql_table_creation.py#L54-L56

But I don't feel too strongly about it.

Reply:
The destructuring assignment is definitely nice for writing tests, and the type annotations save it from being the disaster that tuples used to be. But the nesting adds complexity without much of an upside. This would read and write easier as

```python
workspace, graphs, node_table, edge_table = populated_workspace
```

If you make that change then I'm happy with this.